Production/crab/config/2018/EGamma.txt

```diff
@@ -0,0 +1,4 @@
+lumiMask=/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions18/13TeV/ReReco/Cert_314472-325175_13TeV_17SeptEarlyReReco2018ABC_PromptEraD_Collisions18_JSON.txt
+sampleType=Run2018D applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=False saveGenJetInfo=True
```

Comment:
- `saveGenJetInfo=False`
- prompt is RunD, ReReco are ABC => file names should be swapped.